transport/rest/handler.go

```diff
@@ -22,12 +22,21 @@ type Handler struct {
 	Router          *mux.Router
 	okJson          []byte
 	supportedTokens *service.SupportedTokens
+	contractAddress string
 }
 type genRes struct {
 	StatusText string `json:"statusText"`
 	Status     int    `json:"status"`
 }
 
+// read os env var with default
+func getEnv(key, fallback string) string {
```

Comment:
Could this go in `restutils` please? I'd like to keep handler file short and understandable